Cargo.toml

```diff
@@ -23,8 +23,10 @@ serde_derive = "1"
 serde_json = "1.0"
 rust-crypto = "0.2" 
 sha2 = "0.8.0"
+sha3 = "0.8.2"
```

Comment:
isn't this part of rust-crypto already?

Reply:
Also, why do we need sha3?  We don't use it anywhere else...

Reply:
It is useful for smart contracts that may want to interop with other systems using it, notably ethereum.

Reply:
Removed the redundant sha3 crate